server/src/main/java/server/util/Positions.java

```diff
@@ -0,0 +1,54 @@
+package server.util;
+
+import org.eclipse.lsp4j.Position;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.Comparator;
+
+public class Positions {
```

Comment:
Test case for positions class?

Reply:
The groovy one doesn't seem to have tests. I guess that's no excuse, though